### ISSUE TITLE
Checkpoints

### ISSUE THREAD

**Issue by iquibalh** [mlvlab/Flipped-VQA#5]
I have trained the model with a few datasets, but since the torch seed is changing results are slightly different than those presented in the paper. Could you share the checkpoints to reproduce the results shown in the paper?

**Comment by ikodoh** [comment]
We have uploaded our fine-tuned checkpoint at [here](https://drive.google.com/drive/folders/1XtMZMNW3CRmzvpEVYj29iaUgDFcPUroe?usp=drive_link).
